styles/style.css

```diff
@@ -0,0 +1,72 @@
+/* Minimum desktop screen size should be 375px */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #e5e5e5;
+}
+
+.toolbar {
+  background-color: #fff;
+  padding: 5%;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* for the moment it has no display, this will be set on future commits in the project */
+.navigation {
+  display: none;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url("../images/fondo.png");
```

Comment:
- Nicely done using the background image instead of img tag for the headline section, However, the background image is not responsive on mobile screen sizes for this range of resolution **min 375px**-**max 786px**.

- [ ] I am sorry this wasn't mentioned in the previous review and I highly suggest downloading each design component of the background and setting multiple background images for the headline ensuring that each decoration is in its own position.

**You can check this [CSS Multiple Backgrounds](https://www.w3schools.com/css/css3_backgrounds.asp#:~:text=CSS%20allows%20you%20to%20add,is%20closest%20to%20the%20viewer.)**

## Your Design
![image](https://user-images.githubusercontent.com/68030297/219443525-0596dd20-7b61-4862-b0d8-aa12ce1a060c.png)

## Figma Design
![image](https://user-images.githubusercontent.com/68030297/219444505-62b12180-d1d0-4c3e-81a2-57038702d5f4.png)